shared/actions/login/index.js

```diff
@@ -4,15 +4,14 @@ import {isMobile} from '../../constants/platform'
 import {navigateTo, routeAppend} from '../router'
```

Comment:
the bulk of this PR. we ask for username them we show a list of devices or a password. everything except the gpg stuff should work